ouroboros-consensus-cardano/src/shelley/Ouroboros/Consensus/Shelley/Ledger/Mempool.hs

```diff
@@ -134,7 +134,7 @@ instance ShelleyCompatible proto era
   reapplyTx = reapplyShelleyTx
 
   txsMaxBytes TickedShelleyLedgerState { tickedShelleyLedgerState = shelleyState } =
-      fromIntegral maxBlockBodySize - fixedBlockBodyOverhead
+      maxBlockBodySize - fixedBlockBodyOverhead
```

Comment:
By the way, might be worth leaving a comment here along the lines of: `maxBlockBodySize` is expected to be bigger than `fixedBlockBodyOverhead`, since otherwise there would be an underflow